docs/deployment-process/execution-containers-for-workers/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Execution containers for workers
+description: How to set a step in your deployment/runbook process to run inside a container.
+position: 15
+---
+
+For a [step](/docs/deployment-process/steps/index.md) running on a [worker](docs/infrastructure/workers/index.md) or on the [Octopus Server](docs/infrastructure/workers/built-in-worker.md) that you define in your [deployment processes](/docs/deployment-process/index.md), you can select a container for the step's execution.
```

Comment:
Final tweak to simplify the sentence structure and make the point a bit clearer:
```suggestion
For a [step](/docs/deployment-process/steps/index.md) running on a [worker](docs/infrastructure/workers/index.md) or on the [Octopus Server](docs/infrastructure/workers/built-in-worker.md), you can select a container for the step's execution.
```